src/rules/property-no-unknown/README.md

```diff
@@ -12,35 +12,37 @@ This rule considers properties defined in the [CSS Specifications and browser sp
 
 This rule ignores variables (`$sass`, `@less`, `--custom-property`).
 
+This rule ignores vendor prefixed properties (`-moz-align-self`, `-webkit-align-self`) .
```

Comment:
Let's also mention up here that this can be turned off with the option documented below.